auto-generated/vector-crypto/api-testing/vrev8.c

```diff
@@ -0,0 +1,182 @@
+#include <stdint.h>
+#include <riscv_vector.h>
+
+typedef _Float16 float16_t;
+typedef float float32_t;
+typedef double float64_t;
```

Comment:
What is the point of defining those types which are not used in this file ? (Is this some side effect of the auto generation template)